src/Adapter/NullItems.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\ContentApiBundle\Adapter;
+
+use EmptyIterator;
+use IteratorAggregate;
+use Libero\ContentApiBundle\Exception\ItemNotFound;
+use Libero\ContentApiBundle\Model\ItemId;
+use Libero\ContentApiBundle\Model\ItemList;
+use Libero\ContentApiBundle\Model\Items;
+use Libero\ContentApiBundle\Model\ItemVersion;
+use Libero\ContentApiBundle\Model\ItemVersionNumber;
+use Traversable;
+
+final class NullItems implements IteratorAggregate, Items
+{
+    public function add(ItemVersion $item) : void
+    {
```

Comment:
could throw an exception? Ignore is also a valid choice (`> /dev/null`)

Reply:
Maybe ignoring violates the contract, because `add` followed by `get` on two different implementations has different results

Reply:
Hmm, would need to add an exception to the contract. Could move `add()` to a separate `WriteableItems` interface? (With hopefully a better name...)

Reply:
Maybe don't need a standard exception for failing to add/remove an item. Not sure it's useful to catch it anywhere.

Reply:
Should only be called in error, `BadMethodCallException` would work without necessarily documenting it in the interface